e2etest/bgptests/bgp.go

```diff
@@ -1209,6 +1209,136 @@ var _ = ginkgo.Describe("BGP", func() {
 				false,
 				ipfamily.IPv6))
 
+		table.DescribeTable("A service IP will not be advertised to peers outside the BGPAdvertisement peers list", func(addressRange string, ipFamily ipfamily.Family) {
+			var testContainers []*frrcontainer.FRR
+			for _, c := range FRRContainers {
+				if !strings.Contains(c.Name, "multi") {
```

Comment:
Why this? If you want to pick just two, be explicit and take the first two of the array (after the len check)